flycheck.el

```diff
@@ -6681,15 +6681,13 @@ See URL `https://github.com/ndmitchell/hlint'."
   ((warning line-start
             (file-name) ":" line ":" column
             ": Warning: "
-            (message (one-or-more not-newline)
-                     (one-or-more "\n" (one-or-more not-newline)))
-            line-end)
+            (message (+ (not (any ?\n))))
+            (| ?\n string-end))
```

Comment:
OK, will fix it up shortly.

BTW, please don't merge this yet--I may be on some serious crack, but in my terminal scrollback, I can clearly see that the code was working correctly, but I opened the same file that I'd been using to test just now and it's behaving differently, so I need to get to the bottom of this first (hurrrr durrr).